src/panels/lovelace/cards/hui-map-card.ts

```diff
@@ -437,14 +437,18 @@ class HuiMapCard extends LitElement implements LovelaceCard {
 
   private _getMapEntities(): HaMapEntity[] {
     return [
-      ...(this._configEntities || []).map((entityConf) => ({
-        entity_id: entityConf.entity,
-        color: this._getColor(entityConf.entity),
-        label_mode: entityConf.label_mode,
-        attribute: entityConf.attribute,
-        unit: entityConf.unit,
-        focus: entityConf.focus,
-        name: entityConf.name,
+      ...(this._configEntities || [])
+        .filter(
+          (entityConf) => !this.hass.entities?.[entityConf.entity]?.hidden
```

Comment:
Please move the filter to getAllEntities instead.

Reply:
moved to getAllEntities